apps/etterlatte-tidshendelser/src/main/kotlin/no/nav/etterlatte/tidshendelser/hendelser/HendelseDao.kt

```diff
@@ -33,17 +34,22 @@ class HendelseDao(
                 ?: throw NoSuchElementException("Fant ikke jobb med id $id")
         }
 
-    fun finnAktuellJobb(): List<HendelserJobb> =
+    // Henter jobber for kjoeredato, hvis ikke spesifisert brukes CURRENT_DATE
+    // COALESCE = returns first non-NULL value
+    fun finnAktuellJobb(kjoeredato: LocalDate? = null): List<HendelserJobb> =
```

Comment:
Dette er bare for å gjenbruke eksisterende funksjon, kan skrive en egen om det blir rotete